### DISCUSSION TITLE
Is there any issue having the same job group name for different jobs?

### DISCUSSION THREAD

**Discussion by viveknuna** [quartznet/quartznet#2164] [Q&A]
Is there any issue with having the same job group name for the jobs which are schedules are scheduled at different times like one recurring job is scheduled today and one recurring job was scheduled 10 days back?

**Comment by lahma** [comment]
Groups are more tool for handling batch operations like pausing and resuming. So you could suspend all jobs in group Salesforce to disable an integration for example. The API has many methods that can take GroupMatcher and thus groups help especially in those situations.
